.circleci/config.yml

```diff
@@ -25,12 +33,129 @@ jobs:
       - run:
           name: Root package install
           command: npm ci
+      - save_cache:
```

Comment:
The `save_cache` command here is running after we run `npm ci`, which installs a fresh bunch of `node_modules`. So in this case we're not really benefiting from caching between CircleCI builds where the `package.json` file has not changed. I've also read some articles (e.g. [this one](https://hackernoon.com/circleci-performance-difference-between-cache-and-workspace-5567679c3601)) which suggest that persisting to workspace actually ends up being faster than cache restores and cache saves. When we're going for the caching approach, we also have the overhead of having to checkout the repository on every step too.